test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,27 @@
+contract C {
+  function() external [] s0;
+  function() external [] s1;
+  function copyDynamicallySizedStorageArray() public {
+    s0 = s1;
+  }
+
+}
+contract copyDynamicArrays {
+  C public testContract;
+
+  constructor() {
+    testContract = new C();
+  }
+
+  function tryCopyingDynamicallySizedStorageArray() public returns (uint ret) {
+    try testContract.copyDynamicallySizedStorageArray() {
```

Comment:
If you don't expect an error, you don't have to use `try`

Reply:
Yeah. In case of error this will revert and you'll see `FAILURE` instead of result.